### PR TITLE
add a counter for each url using wasi-keyvalue

### PR DESCRIPTION
**Pull request by Mossaka**

### CODE REVIEW

File: wit/deps/http/error.wit [hunk 1]
```diff
@@ -0,0 +1,10 @@
+interface wasi-cloud-error {
```
Comment (dicej) [inline wit/deps/http/error.wit]: This file (and all the other keyvalue files) are already available under wit/deps/keyvalue.  Can you use those versions instead?

File: subscriber/app.py [hunk 2]
```diff
@@ -42,7 +50,22 @@ async def handle_async(messages: List[Message]):
...
+                        # parse value of bytes to int
+                        value = int(value)
+                        outgoing_value = kv.new_outgoing_value()
+                        kv.outgoing_value_write_body_sync(outgoing_value, bytes(str(value + 1), "utf-8"))
+                        readwrite.set(bucket, url, outgoing_value)
+                    except Exception as e:
+                        outgoing_value = kv.new_outgoing_value()
+                        kv.outgoing_value_write_body_sync(outgoing_value, bytes("1", "utf-8"))
+                        readwrite.set(bucket, url, outgoing_value)
+                        continue
```
Comment (dicej) [inline subscriber/app.py]: What's the reason for the `continue` here?

**Review by dicej — CHANGES_REQUESTED** [review]

File: wit/deps/http/error.wit (hunk 1 above)
Comment (dicej) [reply wit/deps/http/error.wit]: Note that I updated the `wasi-keyvalue` branch of my Spin repo to match this new wit directory structure.

Comment (Mossaka) [reply wit/deps/http/error.wit]: I might have misunderstood but the host implementation of wasi-keyvalue is utilizing keyvalue wits that merged into wasi-http, thus why I copied wasi-keyvalues files to wasi-http.

Comment (Mossaka) [reply wit/deps/http/error.wit]: > Note that I updated the wasi-keyvalue branch of my Spin repo to match this new wit directory structure.

Ohh that makes sense

File: subscriber/app.py (hunk 2 above)
Comment (Mossaka) [reply subscriber/app.py]: Nice find. No reason for it being there

**Review by dicej — APPROVED** [review]

**Pull request merged**